datajoint/condition.py

```diff
@@ -1,7 +1,7 @@
 """ methods for generating SQL WHERE clauses from datajoint restriction conditions """
 
 import inspect
-import collections
+import collections.abc
```

Comment:
Nm, this has actually been handled correctly already. We can leave as is.

```suggestion
import collections
```